### PR TITLE
feat: add docs

### PR DESCRIPTION
**Pull request by duongjitera**

### CODE REVIEW

File: docs/DateTimePicker.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Type: `(value: Date) => void`
+
+Date change handler.
+
+This is called when the user changes the date or time in the UI. It receives the date as parameters.
+
+### `onBlur`
+
+Type: `(event: NativeSyntheticEvent<TextInputEndEditingEventData>) => void`
```
Comment (jefrydco) [inline docs/DateTimePicker.md:23]: This generic value shouldn't be `TextInputEndEditingEventData`, please hover up on the onBlur props then we can know certainly what is the generic value.

File: docs/OTPInput.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Type: `(value: string) => void`
+
+OTP Code change handler.
+
+This is called when the user changes all the data of cells in the UI. It receives the string as parameters.
+
+### `onBlur`
+
+Type: `(event: NativeSyntheticEvent<TextInputEndEditingEventData>) => void`
```
Comment (jefrydco) [inline docs/OTPInput.md:23]: This one as well, 😅

**Review by jefrydco — CHANGES_REQUESTED** [review]
Please help to take a look at the following comments

**Comment by duongjitera** [comment]
thank you @jefrydco I have fixed it, please help again

**Review by jefrydco — APPROVED** [review]
Great work!

**Pull request merged**